System-Architecture/CSC-Overview/Auxiliary-Telescope-CSCs/index.rst

```diff
@@ -15,46 +15,45 @@ As mentioned in the :ref:`Control-Packages-General-CSC-Overview`, it is convenie
 CSCs in this group are strictly associated with the operation and calibration of the AuxTel and cannot be moved between systems.
 
-Although a :ref:`List_of_AT_CSCs` is found at the bottom of this page, it is often useful to associate the CSCs by the control-package it is associated with.
+Although a full list of AuxTel CSCs, indicated by the ``AT`` prefix, are found in the `Master CSC Table <https://ts-xml.lsst.io/#master-csc-table>`__, it is generally most useful to associate the CSCs by the control-package it is associated with, as it done below.
 
 Links are provided to the user guide when available. Interface information for each CSC can be found in the `ts-xml guide <https://ts-xml.lsst.io/>`__.
 
 
 Auxiliary Telescope Control System (ATCS)
 =========================================
 
-The :ref:`ATCS class<Control-Packages-AuxTel-ATCS>` is a high level control package that provides the most user-friendly interface to the Auxiliary Telescope related CSCs related to operating the telescope and dome.
+The :ref:`ATCS class <Control-Packages-AuxTel-ATCS>` is a high level control package that provides the most user-friendly interface to the Auxiliary Telescope related CSCs related to operating the telescope and dome.
 
 The following AT CSCs are associated with that class, but also controllable individually if required:
 
 `ATDome <https://ts-atdome.lsst.io>`__
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-    * The ATDome CSC controls the dome shutter and rotational position.
+    * The `ATDome CSC <https://ts-atdome.lsst.io>`__ controls the dome shutter and rotational position.
 
 `ATDomeTrajectory <https://ts-atdometrajectory.lsst.io>`__
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-    * The ATDomeTrajectory CSC controls how and when the dome moves relative to the telescope position. It essentially monitors the telescope position and trajectory and sends commands to the ATDome CSC when appropriate to do so.
+    * The `ATDomeTrajectory CSC <https://ts-atdometrajectory.lsst.io>`__ controls how and when the dome moves relative to the telescope position. It essentially monitors the telescope position and trajectory and sends commands to the ATDome CSC when appropriate to do so.
 
-`ATPtg <https://ts-atptg.lsst.io>`__
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+ATPtg
+^^^^^
 
     * The AT pointing component CSC converts the celestial position to mount coordinates (altitude and azimuth) and includes an analytical model of the mount to increase pointing and tracking accuracy. It sends position commands to the AT mount control system (ATMCS).
 
 ..  `ATMCS <https://ts-atmcs.lsst.io>`__
 
-ATMCS
-^^^^^
+`ATMCS <https://ts-atmcs.lsst.io/>`__
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
     * The AT mount control system performs the servo control of the telescope mount motors and encoders.
+        During standard on-sky operations, it receives its trajectory and position commands from the pointing component (ATPtg).
```

Comment:
I think the trajectory is computed bu the `ATMCS` itself. The pointing sends demands only. It is up to the mount to figure out how to get there.

Reply:
you're correct... i was thinking PVT vector when i wrote this... which isn't trajectory. Good catch.